### PR TITLE
Set a required ruby version in the Gemfile

### PR DESCRIPTION
**Pull request by dfed**
This PR sets the ruby version in our Gemfile. This should enable engineers working on ARM Macs to utilize `bundle exec pod trunk push` without failing locally. By default, the Gemfile utilizes system Ruby, which unfortunately does not support ARM Macs well.

Cc @paulabrudan

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -11,6 +11,9 @@ jobs:
     name: Pod Lint
     runs-on: macOS-11
     steps:
+      - uses: actions/setup-ruby@v1
+        with:
+          ruby-version: '2.7.6'
```
Comment (dfed) [inline .github/workflows/ci.yml:16]: I'd like to use a newer version of ruby, but this one works and it's the latest supported by the `setup-ruby` script

File: Gemfile [hunk 2]
```diff
@@ -1,3 +1,5 @@
+ruby '2.7.6'
```
Comment (bachand) [inline Gemfile:1]: > By default, the Gemfile utilizes system Ruby, which unfortunately does not support ARM Macs well.

Curious what you mean by this. I'd think/hope the system Ruby shipped on ARM Macs worked OK but maybe that's not the case?

**Review by bachand — APPROVED** [review]
Nice!

File: Gemfile (hunk 2 above)
Comment (dfed) [reply Gemfile:1]: What I know: running `bundle exec pod trunk push` fails with Apple's ruby. But with later versions it works.

With Apple's ruby, we fail loading `ffi`, and the stated solution is to bump the ruby version: see https://github.com/ffi/ffi/issues/922

Comment (dfed) [reply Gemfile:1]: Maybe macOS 13 resolves this issue? But for now this seems reasonable

Comment (bachand) [reply Gemfile:1]: Yep seems very reasonable to enforce a Ruby version 👍

**Pull request merged**